packages/odyssey-react-mui/package.json

```diff
@@ -119,6 +119,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "regenerator-runtime": "^0.14.1",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
This package was last published 6 years ago, but many people are using it, and it's MIT licensed, so we should be fine.

I think this is kind of unnecessary, but once I get Playwright, we'll have it figured out.